lib/generators/framework/USAGE

```diff
@@ -0,0 +1,8 @@
+Description:
```

Comment:
is there some way we could avoid the need for this file by hooking into the usage functionally to explain the usage?

Reply:
I think I'd prefer to leave this file and remove the comment; the generator generator outputs this and it puts your head into the right mode when editing to have it be plain text and not a metadata method with a descriptive string.